cmd/dex/serve.go

```diff
@@ -117,6 +117,9 @@ func runServe(options serveOptions) error {
 		return fmt.Errorf("failed to register gRPC server metrics: %v", err)
 	}
 
+	grpcLogEntry := logrus.NewEntry(logger)
```

Comment:
I wonder if we should mark grpc logs somehow. Like adding a field called subsystem/component/channel.

@nabokihms what do you think?

Reply:
bump